camera_eth_interface: "enp2s0f0"
lidar_eth_interface: "enp2s0f1"

# available interfaces are: enp2s0f0,enp2s0f1,enp2s0f2,enp2s0f3,enp4s0,enp0s31f6